docs/api-input.md

```diff
@@ -4,6 +4,7 @@
 
 -   [clone](#clone)
 -   [metadata](#metadata)
+-   [stats](#stats)
```

Comment:
The `docs/api-*.md` files are autogenerated so this content needs to live in the jsdocs above the new `stats` function.